### ISSUE TITLE
[Enhancement]: Add scope attributes to Cart and Cart Totals table headers for improved accessibility.

### ISSUE THREAD

**Issue by mahfuzurwp** [woothemes/woocommerce#57388]
### Describe the solution you'd like

While auditing a site for accessibility, I noticed that WooCommerce’s cart-related tables contain <th> elements that are missing the scope="col" attribute. This attribute is important for accessibility, as it helps screen readers and assistive technologies associate header cells with their corresponding data cells.

**_I’d be happy to submit a pull request to implement this improvement if it would be helpful._**

**Affected templates:**

1. Cart Table (templates/cart/cart.php)

Example (current):
`<th class="product-name"><?php esc_html_e( 'Product', 'woocommerce' ); ?></th>`

Suggested (with scope):
`<th scope="col" class="product-name"><?php esc_html_e( 'Product', 'woocommerce' ); ?></th>`

2. Cart Totals Table (templates/cart/cart-totals.php)
Example (current):
`<th><?php esc_html_e( 'Subtotal', 'woocommerce' ); ?></th>`

Suggested:
`<th scope="row"><?php esc_html_e( 'Subtotal', 'woocommerce' ); ?></th>`

### Additional context

WCAG Reference:
	•	[H63: Using the scope attribute to associate header cells and data cells in data tables](https://www.w3.org/TR/WCAG20-TECHS/H63.html)

**Comment by gigitux** [comment]
👋 @amberhinds, can you check this issue and share your thoughts? Thanks! 🙇

**Comment by amberhinds** [comment]
Hi, thanks for the tag @gigitux.

Having a scope on table headers is important for row headers or if the table has both row and table headers. 

The [current cart table](https://github.com/woocommerce/woocommerce/blob/ad5cc8d5e8f010d670e6c80bc75db3dc57ce0f55/plugins/woocommerce/templates/cart/cart.php#L4) only has column headers and no row headers. In this instance, the screen reader already reads out the column header when navigating via arrow keys between columns. Here's a video that shows that:

https://github.com/user-attachments/assets/ac454338-894c-4c68-b582-e09ae3336eb2

So, while adding `scope="col"` to the th tags will not hurt anything, in this instance it is not necessary.

**Comment by amberhinds** [comment]
Sorry, I just realized I posted the wrong video for the mentioned code. Here's a video of the cart shortcode table, which is what @mahfuzurwp was referencing in the cart.php template. The column headers are also read out on this table when navigating via the arrow keys between columns.

https://github.com/user-attachments/assets/01a247d8-e470-4373-a730-835fce13c820

What is not read out is the product name when navigating between rows. This is because there's no row headers.

If left as-is, `scope="col"` is unnecessary. If we wanted to improve the accessibility of the shortcode cart then I would recommend the following:

1. Add `scope="col"`
2. Change the `td` tag on the [cell containing the product name](https://github.com/woocommerce/woocommerce/blob/ad5cc8d5e8f010d670e6c80bc75db3dc57ce0f55/plugins/woocommerce/templates/cart/cart.php#L100) to `<th scope="row">`

This will ensure that when going down a column in the table it will read out the name of the product on other cells like the price and quantity.

**Comment by nerrad** [comment]
Hi @mahfuzurwp, if you're interested in doing a PR, that'd be great and we'll be happy to review.

**Comment by mahfuzurwp** [comment]
Hi @nerrad, sure...I'll submit a PR as soon as possible!

**Comment by reasonadmin** [comment]
Would be great to see these fixes due to European Accessibility Act updates in June this year and the need for as a minimum WCAG 2.1 thanks

**Comment by tjcafferkey** [comment]
@mahfuzurwp thanks for being willing to work on this! If you need any assistance with your PR feel free to let me know 😄

**Comment by mahfuzurwp** [comment]
@tjcafferkey Thank you for the offer😄. I got a bit busy, but I'm jumping on it now!

**Comment by opr** [comment]
Closed by [https://github.com/woocommerce/woocommerce/pull/57883](https://github.com/woocommerce/woocommerce/pull/57883)
